build.sbt

```diff
@@ -1,11 +1,6 @@
-name := """sbt-vcpkg"""
-organization := "com.indoorvivants"
-
-sbtPlugin := true
-
 inThisBuild(
   List(
-    organization := "com.indoorvivants",
+    organization := "com.indoorvivants", // TODO : org should probably be com.indoorvivants.vcpkg
```

Comment:
Changing the org would make it easier to get sense of the stats you get from sonatype

Reply:
Yeah, good point